src/scripts/fetch.sh

```diff
@@ -8,14 +8,14 @@ mkdir -p ./generated/plugins
 cp -rf ./src/content/plugins/ ./generated/plugins
 
 # Fetch webpack-contrib (and various other) loader repositories
-node ./src/scripts/fetch_package_names.js "webpack-contrib" "-loader" | node ./src/scripts/fetch_package_files.js "README.md" "./generated/loaders"
-node ./src/scripts/fetch_package_names.js "babel" "babel-loader" | node ./src/scripts/fetch_package_files.js "README.md" "./generated/loaders"
-node ./src/scripts/fetch_package_names.js "postcss" "postcss-loader" | node ./src/scripts/fetch_package_files.js "README.md" "./generated/loaders"
-node ./src/scripts/fetch_package_names.js "peerigon" "extract-loader" | node ./src/scripts/fetch_package_files.js "README.md" "./generated/loaders"
+node ./src/scripts/fetch_package_names.js "webpack-contrib" "-loader" "README.md" "./generated/loaders"
```

Comment:
i guess the last thing i would like changed is the `fetch_package_names.js` which becamse `fetch_packages` after merging with `fetch_package_files.js` functionality, makes sense to me to rename it accordingly

Reply:
Done